app/styles/layout/_aside.scss

```diff
@@ -4,31 +4,90 @@ aside {
   background-color: $grey-lighter;
   color: $black;
   height: auto;
-  overflow-y: scroll; // TODO idk if I should do this? is this ok to do?
+  overflow-y: auto;
   padding: 0;
 
-  p {
-    font-family: $sans-serif;
-    font-weight: bold;
-    letter-spacing: 1px;
-    padding-left: 0.5em;
-    text-transform: uppercase;
+  @include mobile-only {
+    display: grid;
+    grid-template-columns: 100%;
+    grid-template-rows: auto 1fr;
+    grid-gap: 0 0;
+    grid-template-areas:
+      "title"
+      "list";
+
+    top: 0;
+    right: 0;
+    width: 60vw;
+    height: 100vh;
+    position: fixed;
+    overflow: visible;
+    transform: translateX(60vw);
+    transition: transform .4s ease-in-out;
+    &.active { transform: translateX(0); }
+    &.active > #list-detail-switcher:after { content: '>'; }
+
+    #list-detail-switcher {
```

Comment:
I think that since this is interactive, we should make it look interactive- what about using the $secondary-color here instead? You could use lighten/darken to get the right combination. 

What do you think?

Reply:
Sure, it makes sense.

I'll update the color, thanks for the suggestion!

Reply:
Also, what do you think about a border for the sidebar?
(It would be mobile-only)

<img alt="List Border" width="350px" src="https://user-images.githubusercontent.com/39390862/97183969-268c1e80-177d-11eb-926f-4df4f2228e80.png">

Reply:
yeah that looks good 👍